ethereum_history_api/oracles/src/noir/oracles/receiptOracle/encode.ts

```diff
@@ -13,6 +13,17 @@ const MAX_RECEIPT_PROOF_LENGTH = MAX_RECEIPT_LENGTH * MAX_RECEIPT_PROOF_LEVELS;
 export const KEY_LENGTH = 4;
 const BYTES_32_ZERO = '0x0000000000000000000000000000000000000000000000000000000000000000';
 
+export enum OFFSETS {
```

Comment:
This is confusing: two `OFFSETS` enums that differs with one entry. Very easy to make mistake and this one is only used in tests.

Reply:
Thanks. There was a mistake. The other one should only contain receipt fields. Renamed it to `RECEIPT_OFFSETS`